templates/includes/sidebar.html

```diff
@@ -1,4 +1,17 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <p>Username:
+      <a href="{% url "taxi:driver-detail" pk=user.id %}">{{ user.username }}</a>
```

Comment:
The link to the driver's detail view uses `user.id` for the `pk` parameter. Ensure that the `user` object is indeed a `Driver` instance, as this template assumes. If `user` can be another type of user, this could lead to errors.